src/Control/Distributed/Spark/RDD.hs

```diff
@@ -79,6 +89,35 @@ map clos rdd = do
     f <- reflect clos
     call rdd "map" [coerce f]
 
+mapPartitions
+    :: ( Reflect (Closure (Int32 -> Iterator a -> Iterator b)) ty
```

Comment:
This is because `mapPartitions` is implemented in terms of `mapPartitionsWithIndex`. It can be solved I suppose by binding `mapPartitions` instead.